src/k8s/subscriptions/DEV-CSTAR/terraform.tfvars

```
@@ -210,3 +210,13 @@ configmaps_rtdpaymentinstrumentmanager = {
   POSTGRES_SHOW_SQL                                 = "true"
   LOG_LEVEL_RTD_PAYMENT_INSTRUMENT_MANAGER          = "INFO"
 }
+
+
+configmaps_facustomer_override = {
```

```suggestion
configmaps_facustomer = {
```
for naming convention